src/fs-plus.coffee

```diff
@@ -455,36 +455,11 @@ fsPlus =
 
   # Public: Returns true for extensions associated with compressed files.
   isCompressedExtension: (ext) ->
-    _.indexOf([
-      '.bz2'
-      '.egg'
-      '.epub'
-      '.gz'
-      '.jar'
-      '.lz'
-      '.lzma'
-      '.lzo'
-      '.tar'
-      '.tgz'
-      '.war'
-      '.whl'
-      '.xpi'
-      '.xz'
-      '.z'
-      '.zip'
-    ], ext, true) >= 0
+    _.has(COMPRESSED_EXTENSIONS, ext)
 
   # Public: Returns true for extensions associated with image files.
   isImageExtension: (ext) ->
-    _.indexOf([
-      '.gif'
-      '.ico'
-      '.jpeg'
-      '.jpg'
-      '.png'
-      '.tiff'
-      '.webp'
-    ], ext, true) >= 0
+    _.has(IMAGE_EXTENSIONS, ext)
```

Comment:
I think just `IMAGE_EXTENSION.hasOwnProperty(ext)` is fine here (and similarly below). I don't think using `_.has` is much of a help since this is just a simple Object API call.